README.md

```diff
@@ -10,7 +10,7 @@ As an example, training an image classification model from scratch is as simple
 using FastAI
 path = datasetpath("imagenette2-160")
 data = loadtaskdata(path, ImageClassificationTask)
-method = ImageClassification(Datasets.loadclassesclassification("imagenette2-160"), (160, 160))
+method = ImageClassification(Datasets.getclassesclassification("imagenette2-160"), (160, 160))
```

Comment:
This and the change below appear unrelated to the PR. Can you revert them?

Reply:
Sure, but they do cause bugs (namely, loadclasses... gave undefined function) during the tutorial.

Reply:
In that case it's better to open another quick PR. Generally if it's not blocking CI, then it's better to not put unrelated changes in.

Reply:
A PR with just that change would be appreciated, so it can be fixed swiftly 👍